src/features/perps/screens/perps-account-screen/MarketsSection.tsx

```diff
@@ -31,28 +36,49 @@ export const MarketsSection = memo(function MarketsSection() {
           </Box>
         </ButtonPressAnimation>
 
-        <Bleed horizontal="10px" vertical="16px">
-          <ButtonPressAnimation
-            onPress={() => PerpsNavigation.navigate(Routes.PERPS_SEARCH_SCREEN, { type: 'search' })}
-            style={{ padding: 10 }}
-          >
+        <Box flexDirection="row" alignItems="center" gap={8}>
```

Comment:
Noticed the movement is a bit jittery when the context menu is presented. By Volume -> By Price then dismissal will bump over the search button without an animation